cleanlab/classification.py

```diff
@@ -121,10 +122,10 @@ class LearningWithNoisyLabels(BaseEstimator):  # Inherits sklearn classifier
     for their training label.
 
     Given any classifier having the predict_proba() method, an input feature
-    matrix, X, and a discrete vector of labels, s, which may contain
+    matrix, X, and a discrete vector of labels, labels, which may contain
     mislabeling, Confident Learning estimates the classifications that would
     be obtained if the hidden, true labels, y, had instead been provided to
```

Comment:
Elsewhere you sometimes refer to true label as "hidden label" (eg. L279). 
I think this is confusing and would replace these with "true label" instead of "hidden label"